src/components/Toggle/__tests__/toggle.jest.js

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { render, screen, cleanup, fireEvent } from "@testing-library/react";
+import Toggle from "../Toggle";
+
+describe("Toggle Tests", () => {
+  describe("Snapshot Tests", () => {
+    it("renders correctly with empty props", () => {
+      const tree = renderer.create(<Toggle />).toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it("renders correctly when selection defined by default selected (on) ", () => {
+      const tree = renderer.create(<Toggle componentClassName="dummy-class-name" isDefaultSelected />).toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it("renders correctly when selection defined by default selected (off) ", () => {
+      const tree = renderer.create(<Toggle componentClassName="dummy-class-name" isDefaultSelected={false} />).toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it("renders correctly when selection defined by isSelected (on)", () => {
+      const tree = renderer.create(<Toggle componentClassName="dummy-class-name" isSelected />).toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it("renders correctly when selection defined by isSelected (off)", () => {
+      const tree = renderer.create(<Toggle componentClassName="dummy-class-name" isSelected={false} />).toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it("renders correctly when disabled", () => {
+      const tree = renderer.create(<Toggle isDisabled />).toJSON();
+      expect(tree).toMatchSnapshot();
```

Comment:
where are the snapshots themselves? shouldn't they be commited?
did it pass ci?